### ISSUE TITLE
How do I get a terminal size bigger than 80x30

### ISSUE THREAD

**Issue by kerneltime** [redhat-cip/spielbash#1]
I tried to start tmux with a larger size by passing in -x and -y but the output on play still shows it as 80x30.. asciienma on it's own picks up the size of my terminal.. reaching out before spending more time on this, hoping for a quick hack.

**Comment by mhuin** [comment]
Hello,
I am aware of the problem and unfortunately it is still eluding me at the time. I suspect subprocess.Popen is the culprit, but even trying to set the column and row sizes as environment variables didn't work. Please let me know if you have any success with this.

**Comment by kerneltime** [comment]
Thanks for quick response.. all my attempts to trigger tmux to set the width and height have not paid off.. now looking into asciienma itself.. really cool project though! Keep up the good work! 
Ref: https://github.com/vmware/docker-volume-vsphere/wiki/Demos
I have forked the repo and made some changes to not backspace the line output.. I think an alternative can exist to line in the yaml where at the end a cntrl C is sent.. more on that later.

**Comment by JustinAzoff** [comment]
Yeah, subprocess is the problem.  There's no terminal so tmux tries to be helpful by defaulting to a smaller size.  I can't figure out how to turn that off, so this mostly works, still working out some issues

```
-        self.reel = Command('tmux new-session -d -s %s' % self.session_name)
+        self.reel = Command('tmux new-session -d -s %s -x 160 -y 40' % self.session_name)

-        movie = subprocess.Popen(asciinema_cmd % (self.session_name,
-                                                  self.output_file),
-                                 stdout=subprocess.PIPE,
-                                 stderr=subprocess.PIPE,
-                                 shell=True)
+        full_asciinema_cmd = asciinema_cmd % (self.session_name, self.output_file)
+        movie = pexpect.spawn(full_asciinema_cmd, dimensions=(40, 160))
```


fyi, I am also looking at changing the 'backspace over input' thing.  I just changed it to send

    '# ' + line + enter

which works pretty well.  I may end up refactoring the whole thing to remove most of the classes and simplify things, but swapping out subprocess for pexpect fixes most of the immediate problem

**Comment by mhuin** [comment]
Nice find! Would you mind sending a patch for review on https://softwarefactory-project.io when ready? Let me know if you need help with setting up your review.
